.env

```diff
@@ -0,0 +1,28 @@
+#env for database
+DB_HOST=localhost
+DB_DRIVER=postgres
+#set username, password, and database name
+DB_USER=""
+DB_PASSWORD=""
+DB_NAME=""
+DB_PORT=""
+
+#name of the table in the database
+TABLE_NAME=hashfiles
+
+#specific interval of time repeatedly for ticker
+DURATION_TIME=30
+
+#number of running workers in the workerpool
+COUNT_WORKERS=4
+
+PROC_DIR="/proc"
+
+#hashing algorithm for hashing data
+ALGORITHM="SHA256"
+
+#configMap name for hasher
+CONFIG_MAP_NAME_FOR_HASHER=integrity-sum-config
```

Comment:
Did not looks informative to me. Maybe we can add additional comments(like where it is used) to this parameter or change its name?

Reply:
fixed it